backend/src/main/java/com/webappsbusters/parcelmanagement/domain/ParcelDimensionsDto.java

```diff
@@ -0,0 +1,16 @@
+package com.webappsbusters.parcelmanagement.domain;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@NoArgsConstructor
+@Data
+@ToString
+public class ParcelDimensionsDto {
+    int dummywidth;
+    int dummylength;
+    int dummyheight;
+    public int getWidth(Parcel parcel) { return dummywidth = parcel.getWidth(); }
+    public int getLength(Parcel parcel) { return dummylength = parcel.getLength(); }
+    public int getHeight(Parcel parcel) { return dummyheight = parcel.getHeight(); }
```

Comment:
I think those methods are not used, you are using getWidth(), ... generated by Lombok

Reply:
same story as with ParcelDimensions